Models/BotConfig.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Weather_Monitoring_System.Models
+{
+    public class BotConfig
+    {
+        public string BotName { get; set; }
+
+        public bool Enabled { get; set; }
+
+        public decimal Threshold { get; set; }
+
+        public string Message { get; set; }
+
+        public BotConfig(string botName, bool enabled = false, decimal threshold = 0, string message = "")
```

Comment:
I added some comments on previous pull requests. Please consider applying them.
You can address them here directly since this PR contains all the changes and it is opened on the main branch.